apps/src/templates/studioHomepages/ParticipantSections.jsx

```diff
@@ -75,51 +76,56 @@ export default class ParticipantSections extends Component {
     }
 
     return (
-      <ContentContainer heading={heading} description={description}>
-        {liveSections.length > 0 && (
-          <SectionsAsStudentTable
-            sections={liveSections}
-            canLeave={!!isTeacher}
-            updateSections={this.props.updateSections}
-            updateSectionsResult={this.props.updateSectionsResult}
-            isPlSections={this.props.isPlSections}
-          />
-        )}
-        {archivedSections.length > 0 && (
-          <div>
-            <div style={styles.buttonContainer}>
-              <Button
-                className="ui-test-show-hide"
-                onClick={this.toggleViewHidden}
-                icon={viewHidden ? 'caret-up' : 'caret-down'}
-                text={
-                  viewHidden
-                    ? i18n.hideArchivedSections()
-                    : i18n.viewArchivedSections()
-                }
-                color={Button.ButtonColor.gray}
-                style={{margin: 0}}
-              />
-            </div>
-            {viewHidden && (
-              <div>
-                <div style={styles.hiddenSectionLabel}>
-                  {i18n.archivedSections()}
-                </div>
-                <div style={styles.hiddenSectionDesc}>
-                  {i18n.archivedSectionsStudentDescription()}
-                </div>
-                <SectionsAsStudentTable
-                  sections={archivedSections}
-                  canLeave={!!isTeacher}
-                  updateSections={this.props.updateSections}
-                  updateSectionsResult={this.props.updateSectionsResult}
+      <div style={styles.participantSectionsContainer}>
```

Comment:
This whole changed section is just from wrapping it all in a `div`, but Git is highlighting it weird